### PR TITLE
Add Galactifun

### PR DESCRIPTION
**Pull request by Seggan**
Added Galactifun addon

### CODE REVIEW

**Review by Phoenix-Starlight — APPROVED** [review]
Good to go!

File: pages/Addons.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ AlchimiaVitae | Apeiros-46B | <details><summary>expand</summary>An addon that ad
...
 FlowerPower | NCBPFluffyBear | <details><summary>expand</summary>FlowerPower is a magic addon centered around experience and flowers. The base of each recipe requires various flowers, and different items are crafted along the way to make mid to endgame items. A majority of these items function by consuming experience in exchange for a certain effect or buff.</details> | [Readme](https://github.com/NCBPFluffyBear/FlowerPower/blob/master/README.md) | [GitHub](https://github.com/NCBPFluffyBear/FlowerPower/) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/FlowerPower/master/)
+Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inteded to replace the apparently abandoned ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
```
Comment (svr333) [inline pages/Addons.md:66]:
```suggestion
Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon intended to replace the apparently abandoned ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
```

**Review by svr333 — CHANGES_REQUESTED** [review]

File: pages/Addons.md (hunk 1 above)
Comment (WalshyDev) [inline pages/Addons.md:66]:
```suggestion
Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inspired by ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
```

ClayTech is not "apparently abandoned". It had updated forks and the last commit was 2 weeks ago. This has been stated time and time again and this is just misleading.

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: pages/Addons.md (hunk 1 above)
Comment (svr333) [reply pages/Addons.md:66]: gonna mark as resolved as walshy requested changes on the same part

**Review by svr333 — APPROVED** [review]

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**